### PR TITLE
New topic: computer-algebra

### PR DESCRIPTION
**Pull request by WhiteBlackGoose**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

There should be a short explanation about this topic, as quite a lot of developers are unfamiliar with symbolic computations, it might clarify what the related repositories are about.

### CODE REVIEW

File: topics/computer-algebra/index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+aliases: computeralgebra, symbolic-algebra, symbolicalgebra
+display_name: Computer algebra
+short_description: Computer algebra libraries are used for manipulations with mathematical expressions.
+topic: computer-algebra
+wikipedia_url: https://en.wikipedia.org/wiki/Computer_algebra
+---
+Computer (symbolic) algebra systems and libraries are used for manipulations with symbolic mathematical expressions.
+Usually, those libraries are capable of some calculus operations (differentiation, integration, finding the limit, etc.), numeric manipulations.
```
Comment (kenyonj) [inline topics/computer-algebra/index.md:9]:
```suggestion
Usually, those libraries are capable of some calculus operations (differentiation, integration, finding the limit, etc.) and numeric manipulations.
```

seems like an `and` works better here than a comma. WDYT?

**Review by kenyonj — CHANGES_REQUESTED** [review]
Thanks for your contribution. Just one small suggestion.

File: topics/computer-algebra/index.md (hunk 1 above)
Comment (WhiteBlackGoose) [reply topics/computer-algebra/index.md:9]: I guess you're right

**Review by kenyonj — APPROVED** [review]
💯 🍨 Thank you for your contribution! I'll merge this in as soon as CI is 🍏

**Pull request merged**